schemas/string_interpolation/definitions.json

```diff
@@ -37,6 +37,15 @@
       }
     ]
   },
+  "summary": {
```

Comment:
I'm not sure why this is part of this file?. This should be a property of the question, hence update the question types to have this new property?.

Reply:
No idea why I didn't put it in the question definitions, doh